DoLahUI/config.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
+<!DOCTYPE boost_serialization>
+<boost_serialization signature="serialization::archive" version="13">
+<config class_id="0" tracking_level="0" version="0">
+	<storagefile>C:/Users/Seokmin/Documents/Yr3Sem1/cs2103project/DoLah/DoLahUI/calendar.yaml</storagefile>
```

Comment:
Don't commit this Configuration File. It happens when you add all without noticing what you are adding into git.

Use `git add .` with caution.

This file exist because you're switching from another branch which has the gitignore but this branch still has not include the gitignore.